Makefile.dos

```diff
@@ -0,0 +1,30 @@
+# Makefile for MS-DOS using Watcom compiler.
+#
+# wmake -f Makefile.dos
+# - builds the static library xmp_static.lib
+#
+# To disable module depacker functionality:
+#	wmake -f Makefile.dos USE_DEPACKERS=0
+#
+# To disable ProWizard functionality:
+#	wmake -f Makefile.dos USE_PROWIZARD=0
+#
+!ifndef target
+target = static
+!endif
+
+USE_PROWIZARD	= 1
+USE_DEPACKERS	= 1
+
+CC = wcc386
+SYSTEM = dos4g
```

Comment:
In the makefile I have locally, I used `causeway` - IIRC no specific reason (but nasm does that too.)

Reply:
We can also use `dos32a` here (let's do that instead of dos4gw). Actually, we don't really need to specify anything here (because we build static lib only), but whatever.